web/src/api/request.ts

```diff
@@ -0,0 +1,14 @@
+interface ApiRequests {
+  healthCheck: string;
+  fetchSurveyListAll: string;
+  fetchSurvey: string;
+  fetchSurveyListPage: string;
+}
```

Comment:
Could you add a blank line here..? I know it's not a big deal but for consistency of our entire code.